app/tools/jsondb.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+
+class Database:
+    def __init__(self):
+        self.files = {}
+
+    def use(self, name_space):
+        self.name_space = name_space
+
+    def create(self):
```

Comment:
Ok zrobiłaś tak jak mówiłem i jest ok, ale ...
Rozważmy taką sytuację czy metoda use i create jest tak na prawdę potrzebna
jeśli byśmy tworzyli nowa instancję klasy Database w ten sposób users = Databases('users')
to nie musieli byśmy później ani używać metoda use ani create. Oczywiście pod warunkiem że zawarła byś obsłgę tej funkcjonalności w konstruktorze (__init__)